### PR TITLE
outsource the description template into an own file in order to make the...

### PR DESCRIPTION
**Pull request by Bonko**
... code better readable

### CODE REVIEW

File: nagios2trac.py [hunk 1]
```diff
@@ -88,8 +88,14 @@ def read_config(options):
     trac_owner = config.get('Trac', 'ticket_owner')
     TRAC_NOTIFICATIONS = config.get('Trac', 'notifications')
     trac_new_ticket_threshold = int(config.get('Trac', 'new_ticket_threshold'))
+    try:
+        trac_description_template = config.get('Trac', 'description_template')
+        f = open(trac_description_template, 'r')
+        description_template_original = f.read()
+    except:
+        description_template_original = "dummy description"
```
Comment (PirminTapken) [inline nagios2trac.py:96]: Shouldn't it include ":comment_template" so the replace doesn't fail later on?

File: nagios2trac.py [hunk 2]
```diff
@@ -140,39 +146,7 @@ def main(options, args):
...
-    === Post Mortem Analysis ===
-    * How has it been resolved? What was the fix?
-    * How was the issue triggered? What have been the circumstances?
-    * What was affected? (see Incident, Who is already documented there)
-    * How can this situation be avoided in the future? (only relevant if "rule of three" triggered or effect is considered "major" by the team)
-    * Did a known standard solution work? If not: Document it! e.g. in [wiki:NuetzlicheShell], [wiki:AdminHandbuch]
-    * Close the ticket, if no further actions can be derived from it.
-    """
-
+    description_template = description_template_original.replace(':comment_template', comment_template)
```
Comment (PirminTapken) [inline nagios2trac.py:149]: http://docs.python.org/2.6/library/string.html#template-strings might be a more beautiful way in the future

**Comment by PirminTapken** [comment]
What happens when you let this "dummy description" run against trac? does it brick or just create a ticket with "dummy description"?

File: nagios2trac.py (hunk 1 above)
Comment (Bonko) [inline nagios2trac.py:96]: replace does not fail if no string is found

**Comment by Bonko** [comment]
@PirminFix : the latter one

**Comment by PirminTapken** [comment]
@Bonko If this is ok (as this should never happen ;-) ) I'm fine with it :+1:

**Pull request merged**